### DISCUSSION TITLE
Add metatags to Model class

### DISCUSSION THREAD

**Discussion by Zdanc32** [laravel/framework#33335] [General]
Hi, 

I would like to suggest adding meta tags to the Model class. Using methods that cannot be seen is very inconvenient. Like WorkerModel :: select ().

Maybe it doesn't bother other users. But before I found a solution, it took me some time. Then the IDE will display a sugest :D

```
/**
 * @method static select(string[] $array)
 */
abstract class Model implements Arrayable, ArrayAccess, Jsonable, JsonSerializable, QueueableEntity, UrlRoutable
```

Best regards,
Zdanc32

**Comment by netpok** [comment]
You should check out https://github.com/barryvdh/laravel-ide-helper
